### PR TITLE
First animal in the model zoo: cMRI for LVM

### PR DESCRIPTION
**Pull request by lucidtronix**

### CODE REVIEW

File: docker/vm_boot_images/config/pre_requirements.txt [hunk 1]
```diff
@@ -4,11 +4,6 @@ cvxopt
 cvxpy>=1.0
 cython>=0.29
 numexpr
-numpy
 osqp!=0.6.0,!=0.6.1
 pandas>=0.21
-scipy!=1.3.0,>=1.0
-scikit-learn<0.23,>=0.22.0
-pytest
-pytest-cov
```
Comment (paolodi) [inline docker/vm_boot_images/config/pre_requirements.txt:13]: Nice cleanup!!!

File: .github/workflows/python-package.yml [hunk 2]
```diff
@@ -32,7 +32,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install pytest
+        #pip install pytest
```
Comment (paolodi) [inline .github/workflows/python-package.yml:35]: Isn't this required by the command below?

File: docker/vm_boot_images/Dockerfile [hunk 3]
```diff
@@ -33,7 +33,7 @@ RUN apt-get install python3-tk libgl1-mesa-glx libxt-dev -y
 
 # Requirements for the tensorflow project
 RUN pip3 install --upgrade pip
-RUN pip3 install -r pre_requirements.txt
+#RUN pip3 install -r pre_requirements.txt
```
Comment (paolodi) [inline docker/vm_boot_images/Dockerfile:36]: Love the cleanup of duplicate packages! But this gets rid of `pre_requirements.txt` altoghether, right?

File: docker/vm_boot_images/config/tensorflow-requirements.txt [hunk 4]
```diff
@@ -8,7 +8,6 @@ pydicom==1.2.2
 hyperopt==0.1.2
 seaborn==0.9.0
 scikit-image
-scikit-survival
```
Comment (paolodi) [inline docker/vm_boot_images/config/tensorflow-requirements.txt:11]: This one has to go, right? Was it conflicting with some GitHub action?

**Review by paolodi — APPROVED** [review]
Nice work, Sam, this officially opens the zoo! I just installed git lfs and was able to browse model files and plots immediately. I just added a couple of inline comments about package installation. Please double check those and feel free to merge without bouncing through me again!

File: .github/workflows/python-package.yml (hunk 2 above)
Comment (lucidtronix) [reply .github/workflows/python-package.yml:35]: Yeah but it's included in our requirements already.

File: docker/vm_boot_images/Dockerfile (hunk 3 above)
Comment (lucidtronix) [reply docker/vm_boot_images/Dockerfile:36]: Yes. Feel free to resurrect if necessary but tests pass on the cloud still.

File: docker/vm_boot_images/config/tensorflow-requirements.txt (hunk 4 above)
Comment (lucidtronix) [reply docker/vm_boot_images/config/tensorflow-requirements.txt:11]: Yeah wanted numpy 2.20 lol smh.

**Pull request merged**